docs/contributing/marketplace.md

```diff
@@ -11,19 +11,67 @@ This article walks you through the process of contributing content from the Cort
+export const fillDetailsStep = "Fill in additional details, such as, pack name, author, and description.";
+export const signCLAStep = (
+     <p>
+          Sign the <a href="https://github.com/demisto/content/blob/master/docs/cla.pdf">Palo Alto Networks Contributor License Agreement</a>.
+     </p>
+);
+
+<ol start="3">
+     <li>
+          <Tabs
+               defaultValue="create-new"
+               values={[
+                    {label: 'Creating A New Pack', value: 'create-new'},
+                    {label: 'Updating an Existing Pack', value: 'update-existing'},
+               ]}>
+               <TabItem value="create-new">
+               {firstStep}
+               <ul>
+                    <li>{fillDetailsStep}</li>
+                    <li>{githubLoginStep}</li>
+                    <li>{signCLAStep}</li>
+               </ul>
+               </TabItem>
+               <TabItem value="update-existing">
+               {firstStep}
+               <ul>
+                    <li>Select <em>Update Existing Pack</em> from the <em>Select Contribution Mode</em> dropdown.<sup><a href="#footnote1">1</a></sup></li>
+                    <li>Select the pack that you wish to update from the <em>Select Existing Pack</em> dropdown.<sup><a href="#footnote2">2</a></sup></li>
+                    <li>{githubLoginStep}</li>
+                    <li>{signCLAStep}</li>
+               </ul>
+               </TabItem>
+          </Tabs>
+     </li>
+</ol>
+
+
 4. After you submit the form you will be redirected to a page that informs you that pack was received and is being processed.  
+
 5. A GitHub branch will be created in the [xsoar-contrib Content repository fork](https://github.com/xsoar-contrib/content) based on the pull request that will be opened.
+
 6. You will receive an invitation to join the **xsoar-contrib** organization, so the **xsoar-bot** will be able to invite you to a GitHub team and grant you write permissions to the created branch.
 (Each contributor can only modify files in content packs that they contributed).
+
 7. The pull request is created and a reviewer is assigned.
+
 8. You can now modify the files changed in the pull request as part of the review process.
 
+
 :::note
 In order to update the content pack that you contributed, you will need to modify the files in the pull request created in GitHub or close the pull request and create a new contribution.
 :::
+
+:::note Updating Existing Content
+<p><a id="footnote1"><em>1</em></a>: The contribution mode selection dropdown will only appear if content items that were part of your contribution were detected as originating from existing sources (for example, you created a new automation in the UI by clicking "Duplicate Automation").</p>
+<p><a id="footnote2"><em>2</em></a>: The pack options when selecting an existing pack to update will be populated by the packs from which the content items of which your content was duplicated from, originally belong to.</p>
```

Comment:
@Itay4 the language here is _really_ bad and needs improving... I can't think how to succinctly describe this.